doc/api/repl.md

```diff
@@ -497,7 +501,7 @@ changes:
   * `terminal` {boolean} If `true`, specifies that the `output` should be
     treated as a TTY terminal, and have ANSI/VT100 escape codes written to it.
     **Default:** checking the value of the `isTTY` property on the `output`
-    stream upon instantiation.
+    stream upon instantiation and if existent `output.hasColors() === true`.
```

Comment:
This sentence doesn’t parse for me – should there be commas around `if existent`? If that is the idea, should this be ``and, if `output` was passed, ...``?

Reply:
I tried to keep it brief. Looking at it again, it seems I have to require more to actually make clear what happens.